source/utility.cpp

```diff
@@ -785,7 +785,7 @@ Plugin::AMD::ProfileLevel Utility::H264ProfileLevel(std::pair<uint32_t, uint32_t
 Plugin::AMD::ProfileLevel Utility::H265ProfileLevel(std::pair<uint32_t, uint32_t> resolution,
 													std::pair<uint32_t, uint32_t> frameRate)
 {
-	typedef std::pair<uint32_t, uint32_t>             levelRestriction; // Total, Main/Sec, High/Sec
+	typedef std::pair<uint32_t, uint64_t>             levelRestriction; // Total, Main/Sec, High/Sec
```

Comment:
What's the need for this change?

Reply:
Compiler complained about possible data loss since some of the numbers are 64 bit ints, but the largest one `4278190080` should still fit in a uint32 so perhaps it just needs a suffix to tell it that the literal is meant to be a uint.

Reply:
Weird. I'd have been fine with either change.